<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCanvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <widget class="QMenu" name="menu_Pen">
     <property name="title">
      <string>&amp;Pen</string>
     </property>
     <addaction name="action_Red"/>
     <addaction name="action_Green"/>
     <addaction name="action_Blue"/>
    </widget>
    <widget class="QMenu" name="menuChange">
     <property name="title">
      <string>Change</string>
     </property>
     <addaction name="actionGrayscale"/>
     <addaction name="actionBlur"/>
    </widget>
    <addaction name="menu_Pen"/>
    <addaction name="menuChange"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTool"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Red"/>
   <addaction name="action_Green"/>
   <addaction name="action_Blue"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/new.png</normaloff>:/icon/imgs/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/open.png</normaloff>:/icon/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/save.png</normaloff>:/icon/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/exit.png</normaloff>:/icon/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Red">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/red.png</normaloff>:/icon/imgs/red.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Red</string>
   </property>
  </action>
  <action name="action_Green">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/green.png</normaloff>:/icon/imgs/green.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Green</string>
   </property>
  </action>
  <action name="action_Blue">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/blue.png</normaloff>:/icon/imgs/blue.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Blue</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/about.png</normaloff>:/icon/imgs/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
